### PR TITLE
New logo

### PR DESCRIPTION
**Pull request by cecton**
the rendered logo

### CODE REVIEW

File: src/server/views/html/mod.rs [hunk 1]
```diff
@@ -17,7 +17,7 @@ fn render_html<B: Render>(title: &str, body: B) -> Response<Body> {
                 meta charset="utf-8";
                 meta name="viewport" content="width=device-width, initial-scale=1";
                 title { (format!("{} - Deps.rs", title)) }
-                link rel="icon" type="image/png" href="/static/favicon.png";
+                link rel="icon" type="image/png" href="/static/logo.svg";
```
Comment (robjtede) [inline src/server/views/html/mod.rs:20]:
```suggestion
                link rel="icon" type="image/svg+xml" href="/static/logo.svg";
```

**Comment by paolobarbolini** [comment]
Can you update the content type

https://github.com/deps-rs/deps.rs/blob/aa0f4777e26202077be35ca0174943304f5bbad8/src/server/mod.rs#L348

File: src/server/views/html/mod.rs (hunk 1 above)
Comment (paolobarbolini) [inline src/server/views/html/mod.rs:20]: Also...

```suggestion
                link rel="icon" type="image/svg+xml" href="/static/logo.svg";
```

**Review by robjtede — CHANGES_REQUESTED** [review]
References like STATIC_FAVICON_PNG and FaviconPng drop the "png" and it is required that the file be served with the correct content type `image/svg+xml` to be usable as a favicon. (src/server/mod.rs line 349)

File: src/server/mod.rs [hunk 2]
```diff
@@ -346,7 +346,7 @@ impl App {
                 .unwrap(),
             StaticFile::FaviconPng => Response::builder()
                 .header(CONTENT_TYPE, "image/png")
```
Comment (robjtede) [inline src/server/mod.rs:348]:
```suggestion
                .header(CONTENT_TYPE, "image/svg+xml")
```

Comment (cecton) [reply src/server/mod.rs]: Ok that was the last one sorry!!! :D

**Review by robjtede — APPROVED** [review]

**Pull request merged**